lib/puppet/type/sensu_silenced.rb

```diff
@@ -8,6 +8,10 @@
 @summary Manages Sensu silencing
 
 The name of `sensu_silenced` can be used to define `check` and `subscription`.
+If the name used to define a `sensu_silenced` resource does not take the form
```

Comment:
This sentence is very difficult to understand, especially for someone where english is not their first language. It starts with a negation which is also difficult. Could you please reword this documentation. Smaller sentences are easier to understand than big sentences with lots of logic in them.

Reply:
What about now?